src/app/Library/Auth/ResetsPasswords.php

```diff
@@ -68,7 +69,7 @@ protected function rules()
         return [
             'token' => 'required',
             'email' => 'required|email',
-            'password' => 'required|confirmed|min:8',
+            'password' => ['required', 'confirmed', 'min:8', PasswordValidation::defaults()],
```

Comment:
Nope I don't think it will override, it will fail at min:8 as min:8 is earlier in the array

If Password::defaults() defined a min of say 10 then it will fail at Password::defaults as it passes the min:8 rule but fail at the min:10 in Password::defaults()